plugin/route53/route53_test.go

```diff
@@ -34,37 +43,61 @@ func (fakeRoute53) ListResourceRecordSetsPagesWithContext(_ aws.Context, in *rou
 	rrsResponse := map[string][]*route53.ResourceRecordSet{}
 	for _, r := range []struct {
 		rType, name, value, hostedZoneID string
+		aliasDnsName, aliasHostedZoneId  string
```

Comment:
please add a new contained tests for this instead of overloading existing tests. This way we have more, finer grained tests which help debugging and pin pointing mistakes.

Reply:
agreed, will extract the tests in a different file, and revert any changes to the original file